app/src/test/kotlin/compiler/semantic_analysis/Common.kt

```diff
@@ -0,0 +1,18 @@
+package compiler.semantic_analysis
+import compiler.common.diagnostics.Diagnostic
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+fun assertResolutionErrorsEquals(expected: List<Diagnostic>, actual: List<Diagnostic>) {
```

Comment:
I would change name of that method too (in fact, this very name made me notice the naming inconsistency about `ResolutionDiagnostic`).